Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -257,7 +257,8 @@ def load_communities():
 
         self.initComplete = True
 
-    def add(self, tdef, dscfg, pstate=None, initialdlstatus=None, setupDelay=0, hidden=False):
+    def add(self, tdef, dscfg, pstate=None, initialdlstatus=None, setupDelay=0, hidden=False,
```

Comment:
Please use lower-case naming conventions for variable names, so `setupDelay` should be `setup_delay`.